README.rst

```diff
@@ -127,9 +127,25 @@ structured logging format for all microservices developed. Thus, the logging
 from all microservices should be configured like explained here. The use is
 analogous to the previous case, but using the function
 :func:`reconplogger.flask_app_logger_setup` instead, and giving as first argument
-the flask app object. Additional to the previous case, this function replaces
-the flask app and werkzeug loggers to use a reconplogger configured one. The usage
-would be as follows:
+the flask app object.
+
+Additional to the previous case, this function:
+
+- replaces the flask app and werkzeug loggers to use a reconplogger configured one.
+- add to the logs the correlation_id
+- Add before and after request functions to log the request details when the request is processed
+- patch the *requests* library forwarding the correlation id in the any call to other microservices
```

Comment:
"correlation id in the any" => "correlation id in any"

Reply:
done